README.md

```diff
@@ -23,7 +23,7 @@ See [zenoh_cpp_vendor/CMakeLists.txt](./zenoh_cpp_vendor/CMakeLists.txt) for mor
 
 ```bash
 mkdir ~/ws_rmw_zenoh/src -p && cd ~/ws_rmw_zenoh/src
-git clone https://github.com/ros2/rmw_zenoh.git
+git clone https://github.com/ros2/rmw_zenoh.git -b <DISTRO>
```

Comment:
If you make it `-b $ROS_DISTRO` it's just as readable but also copy-pastable.

Reply:
I agree! I was just complying with the other instructions below which are using `<DISTRO>` instead of `$ROS_DISTRO`.

@Yadunund any reason for not having used `$ROS_DISTRO` in the very first version of this README ?